.github/workflows/nightvision.yml

```diff
@@ -19,20 +21,26 @@ jobs:
       
       - name: (2) Install NightVision
         run: |
-            wget -c https://downloads.nightvision.net/binaries/latest/nightvision_latest_linux_amd64.tar.gz -O - | tar -xz; sudo mv nightvision /usr/local/bin/
-                    python -m pip install semgrep --user
+          wget -c https://downloads.nightvision.net/binaries/latest/nightvision_latest_linux_amd64.tar.gz -O - | tar -xz; sudo mv nightvision /usr/local/bin/
+          python -m pip install semgrep --user
 
       - name: (3) Extract API documentation from code
-        run: nightvision swagger-extract ./ -u ${NIGHTVISION_TARGET} --lang spring
+        run: |
+          nightvision swagger extract ./ -t ${NIGHTVISION_TARGET} --lang spring || true
+          if [ ! -e openapi-spec.yml ]; then
+              cp backup-openapi-spec.yml openapi-spec.yml
+          fi
 
       - name: (4) Start the app
         run: docker-compose up -d; sleep 10
 
       - name: (5) Scan the API
-        run: nightvision scan --api ${NIGHTVISION_TARGET} --export-sarif
+        run: |
+          nightvision scan -t ${NIGHTVISION_TARGET} -a ${NIGHTVISION_APP} --auth ${NIGHTVISION_AUTH} > scan-results.txt
+          nightvision export sarif -s "$(head -n 1 scan-results.txt)" --swagger-file openapi-spec.yml
```

Comment:
It is a bit awkward to separate this into two commands can we add the `--export-sarif` command back in?